content/docs/atom-guide/dev-guide/shaders/azsl/_index.md

```diff
@@ -422,14 +422,14 @@ An `int` option requires a range attribute to specify the minimum and maximum ra
      
     [[range(3, 16)]] // This integer option accepts values between 3 and 16 (both ends included).  
     option int o_numberOfTaps;
-  
-Even though the Shader Variant Options are declared globally, they are actually encoded in a single array of bits as a member variable of one, and only one, SRG.  
-To learn more about how Shader Variant Options are encoded when compiled, please read: [Shader Variant Options & The Fallback Key.](shader-variants-fallback-key)  
+```
+Even though the Shader variant options are declared globally, they are actually encoded in a single array of bits as a member variable of one, and only one, SRG.  
```

Comment:
Switch to singular "a Shader variant option" to clarify that each shader variant option is encoded in a single array, rather than all collectively. 
Other suggestions to improve flow and clarify details.
```suggestion
Even though a Shader variant options is global, it's encoded in a single array of bits because it's a member of only the SRG that defines it. To learn more about how AZSLc encodes Shader variant options during compilation, read [Shader variant options & The Fallback Key](shader-variants-fallback-key).
```

Reply:
The rewording is not accurate. Leaving as is.